cli/azd/extensions/microsoft.azd.extensions/internal/resources/languages/python/commands/context_command.py

```diff
@@ -0,0 +1,101 @@
+import json
+import asyncio
+import sys
+import os
+import grpc
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "generated_proto")))
+from azd_client import AzdClient
+from google.protobuf import empty_pb2
+import environment_pb2
+import deployment_pb2
+
+
+class ContextCommand:
+    def __init__(self, azd_client: AzdClient):
+        self.azd_client = azd_client
+
+    async def execute(self):
+        try:
+            # User Config
+            config_response = await asyncio.to_thread(
+                self.azd_client.user_config.Get, empty_pb2.Empty()
+            )
+            if config_response and config_response.found:
+                print("User Config")
+                config = json.loads(config_response.value.decode("utf-8"))
+                print(json.dumps(config, indent=4))
+
+            # Project
+            project_response = await asyncio.to_thread(
```

Comment:
Are there easier ways to call into GRPC services from python without having to have these `asyncio.to_thread(...)` functions used all over?